package.json

```diff
@@ -81,7 +81,7 @@
     "electron-dl": "3.5.2",
     "electron-find": "1.0.7",
     "electron-react-titlebar": "1.2.1",
-    "electron-updater": "6.2.1",
+    "electron-updater": "6.4.0-alpha.0",
```

Comment:
@Alphrag  - should we upgrade to an alpha version? Would that be stable for our needs?

Reply:
For `electron-updater`, the only commit on the alpha version is to do with the auto-update of pacman, so we can go to the stable version if you prefer, but that should not bother us. The only reason I decided to put this version, is to match with the electron-builder introducing the pacman possible update, so should we want to add it to our possible targets, they would work straight away together without having to figure out whether one of them needs updating first.